vendor/github.com/docker/distribution/vendor/README.md

```diff
@@ -0,0 +1,22 @@
+The packages in this directory represent dependencies that differ
```

Comment:
It's a brief description from 41f478a37e7648662945c464511f5a7f285afdd0 about why this vendor/ subdirectory exists and how one can recreate it. I found it useful when I was doing rebase. And yes, it another proof that we constantly loose changes related to distribution.